inc/class-googleanalytics.php

```diff
@@ -0,0 +1,422 @@
+<?php
+/**
+ * @author  Pressbooks <code@pressbooks.com>
+ * @license GPLv3 (or any later version)
+ */
+
+namespace Pressbooks;
+
+class GoogleAnalytics {
+
+	/**
+	 * @var null|GoogleAnalytics
+	 */
+	protected static ?GoogleAnalytics $instance = null;
+
+	public static string $is_allowed_option = 'ga_mu_site_specific_allowed';
+
+	public static string $google_id_v3_option = 'ga_mu_uaid';
+
+	public static string $google_id_v4_option = 'ga_4_mu_uaid';
+
+	private string $v3_input_legend;
+
+	private string $v4_input_legend;
+
+	private string $v3_input_label;
+
+	private string $v4_input_label;
```

Comment:
This could be replaced by an `array` I think. 

```php
$ga_versions = [
  'v3' => [
    'label' => ...,
    'legend' => ...,
    'id_option' => ...,
  ],
  'v4' => [
    'label' => ...,
    'legend' => ...,
    'id_option' => ...,
  ]
]
```

We could replace the methods to return data from the array instead of having multiple variables declared. What do you think?